spec/integration/provider/package_spec.rb

```diff
@@ -15,6 +15,13 @@
         expect { pkg.provider.install }.to raise_error
       end
 
+      # TODO: this fails on Windows if the active code page is IBM437 with
```

Comment:
This no longer fails after last commit that translates registry strings (key names, value names and actual values) to UTF8... but still need some sort of test here.